props/saber_fett263_buttons.h

```diff
@@ -411,9 +411,15 @@ SaberFett263Buttons() : PropBase() {}
 #endif
         if (!swinging_) {
           swing_blast_ = false;
-#ifndef FETT263_BM_DISABLE_OFF_BUTTON
+#ifdef FETT263_BM_DISABLE_OFF_BUTTON
+        if (!battle_mode_) {
           Off();
           saber_off_time_ = millis();
+        }
+#endif
+#ifndef FETT263_BM_DISABLE_OFF_BUTTON
```

Comment:
#else

Reply:
Or, you could write this as:

#ifdef FETT263_DISABLE_OFF_BUTTON
      if (!battle_mode)
#endif 
      {
        Off();
        saber_off_time_ = millis();
       }

Reply:
Fixed, thanks.